Singleton_Design_Pattern/default_private_constructor/Singleton.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+class wifimanager{
```

Comment:
it should be WifiManager not wifimanager , follow this https://www.thoughtco.com/using-java-naming-conventions-2034199